src/Domain/ContentRenderer/CodeRenderer.php

```diff
@@ -47,4 +47,22 @@ private function getLanguageClasses(): array {
 		return [ 'language-' . htmlspecialchars( $this->language ) ];
 	}
 
+	/**
+	 * @return (string|string[])[]
```

Comment:
`string[]` or `array<string, string>` if you want to be more detailed

Reply:
string[] failed the test when I first used it. array<string, string> also fails. The above I've used is what was suggested by psalm.

![image](https://github.com/ProfessionalWiki/ExternalContent/assets/5208120/c8f258de-4177-45c2-892a-b09168c272ef)

Reply:
Right, getWrapperClasses returns string[]